### PR TITLE
add the option to make a newline in the image

### PR DESCRIPTION
**Pull request by dberthault**

### CODE REVIEW

File: qollage/qollage/qollage.pyi [hunk 1]
```diff
@@ -46,6 +47,8 @@ def draw_circuit(
          * initialization_mode (str, optional): What to display at the beginning of the circuit:\n
              - "state" for "|0>". Used if None. \n
              - "qubit" for "q[n]".\n
+         * max_circuit_length (Optional(int)): The maximum number of gates per qubit before going to a new line.
+             Default None will never do a newline.
```
Comment (dgupta-hqs) [inline qollage/qollage/qollage.pyi:51]: Default `None` does not create a new line

File: qollage/qollage/qollage.pyi [hunk 2]
```diff
@@ -74,6 +78,8 @@ def save_circuit(
...
```
Comment (dgupta-hqs) [inline qollage/qollage/qollage.pyi:82]: Default `None` does not create a new line

File: roqollage/tests/interface.rs [hunk 3]
```diff
@@ -40,8 +40,8 @@ use test_case::test_case;
 #[test_case(Operation::from(FSwap::new(0, 1)); "FSwap")]
 #[test_case(Operation::from(MeasureQubit::new(0,"ro".to_owned(), 0)); "MeasureQubit")]
 #[test_case(Operation::from(Toffoli::new(0, 1, 2)); "Toffoli")]
-#[test_case(Operation::from(GateDefinition::new(vec![Operation::from(RotateX::new(0, CalculatorFloat::from("theta"))), Operation::from(RotateX::new(1, CalculatorFloat::PI))].into_iter().collect(), "test_gate".to_owned(), vec![0, 1], vec!["theta".to_owned()])); "GateDefinition")]
-#[test_case(Operation::from(CallDefinedGate::new("test".to_owned(), vec![0, 1], vec![CalculatorFloat::from("3.14")])); "CallDefinedGate")]
+// #[test_case(Operation::from(GateDefinition::new(vec![Operation::from(RotateX::new(0, CalculatorFloat::from("theta"))), Operation::from(RotateX::new(1, CalculatorFloat::PI))].into_iter().collect(), "test_gate".to_owned(), vec![0, 1], vec!["theta".to_owned()])); "GateDefinition")]
```
Comment (dgupta-hqs) [inline roqollage/tests/interface.rs:43]: Why has this been commented out?

**Review by dgupta-hqs — COMMENTED** [review]
There is one comment that needs to be addressed. Looks good otherwise

File: roqollage/tests/interface.rs (hunk 3 above)
Comment (dberthault) [reply roqollage/tests/interface.rs]: It was moved in a feature, I created a test for it now

**Review by dgupta-hqs — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,12 @@
 
 This changelog track changes to the qollage project starting at version v0.1.0
 
+## 0.4.0
+
+* Fixed an issue where qollage wouldn't be able to convert some circuits.
+* Added the `max_circuit_length` parameter to define the maximum circuit length on the image.
+* Fixed a bug where control wires would sometimes overlap with gates.
+* Fixed a bug where the font file couldn't be downloaded due to wrong TLS certificates.
 ## 0.3.0
```
Comment (dberthault) [inline CHANGELOG.md:11]:
```suggestion
* Fixed a bug where the font file couldn't be downloaded due to wrong TLS certificates.

## 0.3.0
```

**Pull request merged**